src/azure-cli/azure/cli/command_modules/appservice/_validators.py

```diff
@@ -184,3 +190,15 @@ def validate_ip_address(namespace):
                 return
 
         raise CLIError('Invalid IP address')
+
+
+def _validate_ip_address_existance(cmd, namespace):
+    resource_group_name = namespace.resource_group_name
+    name = namespace.name
+    slot = namespace.slot
+    from ._appservice_utils import _generic_site_operation
+    configs = _generic_site_operation(cmd.cli_ctx, resource_group_name, name, 'get_configuration', slot)
+    access_rules = configs.ip_security_restrictions
+    is_exists = [(lambda x: x.ip_address == namespace.ip_address)(x) for x in access_rules]
```

Comment:
Is it necessary to use lambda here?

Reply:
Yes with out lambda it is not yielding the output.